test/paginator_test.py

```diff
@@ -0,0 +1,49 @@
+from seam import Seam
+
+
+def test_paginator_first_page(seam: Seam):
+    paginator = seam.create_paginator(seam.connected_accounts.list, {"limit": 2})
+    connected_accounts, pagination = paginator.first_page()
+
+    assert isinstance(connected_accounts, list)
+    assert len(connected_accounts) == 2
+    assert pagination is not None
+    assert pagination.has_next_page is True
+    assert pagination.next_page_cursor is not None
+    assert pagination.next_page_url is not None
+
+
+def test_paginator_next_page(seam: Seam):
+    paginator = seam.create_paginator(seam.connected_accounts.list, params={"limit": 2})
```

Comment:
Possible to just do this?
```suggestion
    paginator = seam.create_paginator(seam.connected_accounts.list, {"limit": 2})
```

Reply:
78e49d08045fe727f5a8bb1bf66a0363a5eef639